src/test/java/seedu/address/model/person/BirthdayTest.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class BirthdayTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Birthday(null));
+    }
+
+    @Test
+    public void constructor_invalidBirthday_throwsIllegalArgumentException() {
+        String invalidBirthday = "";
+        assertThrows(IllegalArgumentException.class, () -> new Birthday(invalidBirthday));
+    }
+
+    @Test
+    public void isValidBirthday() {
+        // null address
+        assertThrows(NullPointerException.class, () -> Birthday.isValidBirthday(null));
+
+        // invalid birthday
```

Comment:
Should remove these if its not being used as test cases, idt should leave as comment